physionet-django/project/serializers.py

```diff
@@ -0,0 +1,15 @@
+# from user.models import User
+from rest_framework import serializers
+
+from .models import PublishedProject
```

Comment:
i think our preference is for absolute imports.

Reply:
Ok, Understood - will keep in mind.